src/scene/text-html/HTMLTextPipe.ts

```diff
@@ -34,6 +34,19 @@ export class HTMLTextPipe implements RenderPipe<HTMLText>
     constructor(renderer: Renderer)
     {
         this._renderer = renderer;
+        this._renderer.runners.resolutionChange.add(this);
+    }
+
+    public resolutionChange()
+    {
+        for (const i in this._gpuText)
+        {
+            const gpuText = this._gpuText[i];
+            const text = gpuText.batchableSprite.renderable as HTMLText;
+
+            text._resolution = text._autoResolution ? this._renderer.resolution : text.resolution;
```

Comment:
mini optimisation - we only need to update the ones with text._autoResolution set to true.

```
if(text._autoResolution)
{
    // will trigger on viewUpdate
    text.resolution = this.renderer.resolution
}
```